subprojects/docs/src/docs/userguide/javaPlugin.adoc

```diff
@@ -703,6 +703,60 @@ The TestNG framework has a quite similar concept. In TestNG you can specify diff
             </sample>
 ++++
 
+[[migrate_to_junit5]]
+==== Migrate to JUnit 5
```

Comment:
Wondering if we should explicitly spell out limitations in this section e.g. nested test classes are not discovered.

Reply:
Good idea! Added a note section.